web/src/components/troubleshoot/SupportBundleList.tsx

```diff
@@ -137,15 +170,18 @@ class SupportBundleList extends React.Component {
     if (downstream) {
       if (supportBundles?.length) {
         bundlesNode = supportBundles
-          .sort((a, b) => new Date(b.createdAt) - new Date(a.createdAt))
+          .sort(
+            (a, b) =>
+              new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime()
```

Comment:
Date can return many types. `getTime` returns one type that you can do arithmetic with

Reply:
good to know!